### PR TITLE
src/install: remove trailing whitespace from RAUC_SLOTS and RAUC_TARGET_SLOTS

### PR DESCRIPTION
**Pull request by jluebbe**
The whitespace is not relevant for shell loops, but can be confusing
nevertheless. Remove it.

### CODE REVIEW

File: src/install.c [hunk 1]
```diff
@@ -545,8 +545,8 @@ static void prepare_environment(GSubprocessLauncher *launcher, gchar *update_sou
 		g_clear_pointer(&varname, g_free);
 	}
 
-	g_subprocess_launcher_setenv(launcher, "RAUC_SLOTS", slotlist, TRUE);
-	g_subprocess_launcher_setenv(launcher, "RAUC_TARGET_SLOTS", targetlist, TRUE);
+	g_subprocess_launcher_setenv(launcher, "RAUC_SLOTS", g_strchomp(slotlist), TRUE);
+	g_subprocess_launcher_setenv(launcher, "RAUC_TARGET_SLOTS", g_strchomp(targetlist), TRUE);
```
Comment (ejoerns) [inline src/install.c:549]: Wouldn't it be more straightforward to remove the trailing whitespaces in the corresponding `g_strdup_printf()` statements?

Comment (jluebbe) [reply src/install.c]: We need the spaces between the values, so how would that work when removing them from the `g_strdup_printf()`? I've pushed a version which uses GString.

Comment (ejoerns) [reply src/install.c]: True. I have ignored the while loop :)

I like the solution, looks much cleaner now than the original one with the tmp string. :+1:

**Review by ejoerns — APPROVED** [review]
:+1:

**Comment by zyga** [comment]
Looks good

**Pull request merged**